assets/src/edit-story/components/canvas/layout.js

```diff
@@ -112,6 +121,21 @@ const PageAreaFullbleedContainer = styled(Area).attrs({
   }
 `;
 
+const ZOOM_PADDING = '72px';
```

Comment:
Is this related to the `horizontalPaddingForFit` above?

Reply:
Actually not, not. This is the inner spacing added inside the scrollable element to make edges "stand out". And I'm checking with Sam if this is the proper value here.